app/assets/stylesheets/sections/deploy.scss

```diff
@@ -0,0 +1,56 @@
+$dropbox_color: #1081de;
+$zip_color:     #a58463;
```

Comment:
Name of variable `zip_color` should be written in all lowercase letters with hyphens instead of underscores